packages/e2e-playwright/models/issuer-list.ts

```diff
@@ -28,6 +28,23 @@ class IssuerList {
     async selectHighlightedIssuer(issuerName: string) {
         await this.highlightedIssuerButtonGroup.getByRole('button', { name: issuerName }).click();
     }
+
+    async typeToFilterTerm(filter: string) {
+        await this.selectorCombobox.focus();
```

Comment:
I wonder if there is any difference between doing `.focus()` against `.click()` here. I believe `click` emulates the 'real user behavior' more than focusing on it programatically. Any thoughts on this?

Reply:
My objective using focus, instead of click was to emulate keyboard/screen reader movement. We already test flows with `.click()` does aims to cover a different use case.